UnityProject/Assets/Scripts/Systems/Occupations/Occupation.cs

```diff
@@ -126,4 +126,12 @@ public enum OccupationDifficulty
 	[SerializeField] private bool isTargeteable=true;
 
 	public bool IsTargeteable => isTargeteable;
+
+	/// <summary>
+	/// Set the maximum simultaneous players with this occupation. Set to -1 for unlimited.
+	/// </summary>
+	public void SetLimit(int newLimit)
+	{
+		limit = newLimit;
```

Comment:
just realized something. This won't work as expected because changes to scriptable objects are persistent and will persists through game sessions.

Reply:
Ah. Can't have that.

Reply:
a way around this problem is to load values from the scriptable object into whatever class that uses it and only then modify as you please, the value from the class not the scriptable object.

Reply:
That won't be terribly trivial. I'll take a look at it tomorrow.